### ISSUE TITLE
Group does not exist

### ISSUE THREAD

**Issue by curryhoden** [schollz/find3#138]
Hey and good evening,

im back here after a few years and trying your version3.

After i installed the app and put in my data (family, device, room), i started learning.

5 mins after leaning starts, i wanted to check the status in the dashboard.

But there are no data. 
Latest app version from store.
Tried to login @ cloud.indoor...

Family 'xyz' does not exist. Follow these instructions to get started.

**Comment by schollz** [comment]
Is it still not working for you?

**Comment by schollz** [comment]
Can you explain in detail?

**Comment by schollz** [comment]
Server was down, try again

**Comment by dgraposo** [comment]
Sorry, but I think that the problem is not that. If I try in my docker image server the same thing occur. The only way that I can create a group is by using the API.

**Comment by schollz** [comment]
Are you sure fingerprints are being generated from your android?

**Comment by curryhoden** [comment]
In my case:
i used android pie and the newest version of the app.

I tried a lot of times but there was no way of logging in to my location.

**Comment by dgraposo** [comment]
Yes, the fingerprints are being generated. I also follow all the instructions. We have a small project that use find2. We change all the database to find3 using the API and all work. Only the Android application is not working.

**Comment by curryhoden** [comment]
UPDATE:

Having the same:
-Android Device (xiaomi)
-with same (Find3) app version (latest from android store)
-same Android version incl. latest updates ( Pie with april updates)
- in the same wifi (2.4 / 5 Ghz)
- with the same family/location settings in Find3 (different device of course)
-same wifi and app secure settings (allow wifi/gps/rights of find3 app/etc)

Device 1: 
is uploading its data fine. learning and tracking are directly shown at the dashboard

Device 2: (remember: same family, same location, different device)
App starts learning with text: "learning: App is running : x sec"
But it NEVER shows an upload, like "12 wifi beacon uploaded"
And the dashboard/database is never retrieving data.

We restarted both phones, drop the database, creating a new family, trying add to existing.
SAME PROBLEM!

**Comment by pgodinic** [comment]
@curryhoden did you solve this? If yes how?

**Comment by IoTThinks** [comment]
To use esp client. More reliable.
Android seems to block or throttle something.
